### ISSUE TITLE
Not working with SugarCRM 6.4.x

### ISSUE THREAD

**Issue by rahul286** [cognitom/ModernAqua-Theme-for-SugarCRM#1]
Hi,

I manage to get ModerAqua theme running on SugarCRM 6.4 RC build. (after making changes to manifest.php & themedef.php)

I saw some major issues with missing icons, broken links. 

Do you have any plans to make this compatible with SugarCRM 6.4.x ?

Thanks,
-Rahul

**Comment by ernierasta** [comment]
Hi,
can you test it now? It is now working on 6.3.1, so it should on 6.4 too.

**Comment by rahul286** [comment]
Hey thanks for update! 
I will definitely test next week as I am traveling at the moment.

**Comment by dauclair** [comment]
I have tested somewhat thoroughly on 6.4.0 and 6.4.1 and it appears everything is working. Bravo!

There are some minor css glitches that ernierasta has mentioned in a separate Issue, which I think can be easily fixed: https://github.com/cognitom/ModernAqua-Theme-for-SugarCRM/issues/6

**Comment by rahul286** [comment]
I found some issues with Calendar. Pop-up to add calendar event is completely broken.

**Comment by ernierasta** [comment]
Hi Rahul, could you be more specific please. Please provide:
SugarCRM version:
How to reproduce:

Thanx.

Leszek

**Comment by rahul286** [comment]
@ernierasta 
SugarCRM v6.4.2
Go to `Modules >> Calendar`
You will see formatting for calendar is not proper. 
cal.css file need fixing on line #37
`
    #cal-grid{
       background: white; 
    }
`

There are some more fixes for cal.css
Should I fork relavant CSS and create a patch here on github? 

Also, when you click on any calendar cell to add event popup is not positioned properly.

**Comment by TheNextWave** [comment]
We're having some issues with it as well- with the dark gray background with black text options- in the campaign module.
It's also seemingly losing some links- when it displays first with a list of options for sending mail, sending test- - that then collapse- and don't appear. It's as if the links appear- wait for buttons- that don't display.

**Comment by ernierasta** [comment]
Hi and thanx for report. I know about that and I am not going to fix it. I am really out of free time now. I will probably make branches for every sugarcrm release we want to support in future (this will be marked 6.3). In SugarCRM 6.5 this theme will be broken even more, so if I will be able to spend some time on this project I will try to prepare AquaModern for 6.5 (no promises it not depend on me). But if anyone want to fix it for 6.4 - no problem please fork it and make pull request.

**Comment by TheNextWave** [comment]
That's really depressing- since it makes Sugar bearable.
6.5 promises a really new slick theme- which isn't showing up in the CE edition.
It's too bad this didn't become the default Sugar theme you did an amazing job of improving a crap ui
You shoudl be really proud of what you did accomplish. Thanks.
We'll have to look at going back to 6.3 or so.

**Comment by ernierasta** [comment]
I agree with you, but i am not theme author - Tsutomu Kawamura is our hero ;-). I just make it work with 6.3 and try to fix few glitches after that. Even 6.3 is not yet perfect (i hope to make few fixes for 6.3 later), but it is very usable I think.
